interface/js/app/rspamd.js

```diff
@@ -274,6 +274,19 @@ function ($, D3pie, visibility, NProgress, tab_stat, tab_graph, tab_config,
             }, 1000);
         });
 
+        $.ajax({
+                type: 'GET',
+                url: 'stat',
+                success: function () {
+                    saveCredentials({}, 'nopassword');
```

Comment:
I think it should be enough just to save credentials as there is nothing to hide yet. So we do not need the lines below this comment. And in my opinion it would be logical to do this at the beginning of `ui.connect` function instead of `ui.setup`.

Reply:
hi @moisseev really appreciated your input. I tested in `ui.connect` but I need to hide and displayUI
it looks

```
    ui.connect = function () {


            $.ajax({
                type: 'GET',
                url: 'stat',
                success: function () {
                    saveCredentials({}, 'nopassword');
                    $(dialog).hide();
                    $(backdrop).hide();
                    displayUI();                     
                },
           });
```
do you want I commit it ?